packages/hix/lib/Hix/Bootstrap.hs

```diff
@@ -282,6 +291,20 @@ bootstrapFiles conf = do
   where
     paths = traverse (noteBootstrap "File path error" . parseRelFile)
 
+
+initGitAndFlake :: M ()
+initGitAndFlake = do
+  AppResources { cwd } <- M ask
+  let gitEnv = GitEnv { backend = GitBackend $ gitExec mempty, repo = cwd }
```

Comment:
I know I said to not bother, but now that I look at it again it seems straightforward to use:

```haskell
runGitNative cwd "init new project" \ git -> do
  git.cmd_ ["add", "."]
```

That also obviates `isGitFailure`, since you can use `isLeft` on the result of `git.cmd'`

Reply:
oh yeah yeah! nicer indeed! tbh I didn't fully understand this git interface yesterday. It is pretty cool now that I see it